pkg/hub/manager/manager.go

```diff
@@ -163,29 +184,38 @@ func Start(meshClient client.Client, ctx context.Context) {
 		os.Exit(1)
 	}
 
-	workerSliceGwRecyclerEventRecorder := events.NewEventRecorder(mgr.GetEventRecorderFor("workerslicegwrecycler-controller"))
+	// workerSliceGwRecyclerEventRecorder := mevents.NewEventRecorder(meshClient, mgr.GetScheme(), ossEvents.EventsMap, mevents.EventRecorderOptions{
+	// 	Cluster:   ClusterName,
+	// 	Project:   ProjectNamespace,
+	// 	Component: "workerslicegwrecycler-controller",
+	// 	Namespace: controllers.ControlPlaneNamespace,
+	// 	Version:   utils.EventsVersion,
+	// 	Slice:     utils.NotApplicable,
+	// })
 	if err := (&workerslicegwrecycler.Reconciler{
 		MeshClient:            meshClient,
 		Log:                   ctrl.Log.WithName("controllers").WithName("workerslicegwrecycler"),
 		Scheme:                mgr.GetScheme(),
 		Client:                mgr.GetClient(),
 		WorkerGWSidecarClient: workerGWClient,
 		WorkerRouterClient:    workerRouterClient,
-		EventRecorder:         workerSliceGwRecyclerEventRecorder,
+		EventRecorder:         &spokeSliceEventRecorder,
 	}).SetupWithManager(mgr); err != nil {
 		log.Error(err, "could not create controller")
 		os.Exit(1)
 	}
 
-	spokeClusterEventRecorder := mevents.NewEventRecorder(meshClient, mgr.GetScheme(), ossEvents.EventsMap, mevents.EventRecorderOptions{
-		Cluster:   ClusterName,
-		Project:   ProjectNamespace,
-		Component: "worker-operator",
-		Namespace: controllers.ControlPlaneNamespace,
-	})
+	// spokeClusterEventRecorder := mevents.NewEventRecorder(meshClient, mgr.GetScheme(), ossEvents.EventsMap, mevents.EventRecorderOptions{
+	// 	Cluster:   ClusterName,
+	// 	Project:   ProjectNamespace,
+	// 	Component: "worker-operator",
+	// 	Namespace: controllers.ControlPlaneNamespace,
+	// 	Version:   utils.EventsVersion,
+	// 	Slice:     utils.NotApplicable,
+	// })
 	clusterReconciler := hubCluster.NewReconciler(
 		meshClient,
-		spokeClusterEventRecorder,
+		&spokeSliceEventRecorder,
```

Comment:
Can you please change this spokeSliceEventRecorder to workerSliceEventRecorder

Reply:
done